### ISSUE TITLE
I have created CentOS and Oracle Linux packer scripts

### ISSUE THREAD

**Issue by elico** [GNS3/gns3-registry#716]
Hey,

In my repo:
https://github.com/GNS3/gns3-registry/compare/master...elico:gns3-registry:master

I added 2 packer scripts for both CentOS 8 stream and Oracle Enterprise linux 8 based on their cloud images.

Would it be possible to add these into the registry repo?
I believe it would help some admins to re-pack these OS for local usage.
The benefit is that the updates for the OS is done only once in a generic way compared to upgrading each instance in a lab.

**Comment by grossmj** [comment]
Yes please, can you create a new PR? thanks

**Comment by elico** [comment]
@grossmj thanks.
